pkg/server/test/list_objects.go

```diff
@@ -530,6 +533,7 @@ func BenchmarkListObjectsWithConcurrentChecks(b *testing.B, ds storage.OpenFGADa
 		Logger:           logger.NewNoopLogger(),
 		Meter:            telemetry.NewNoopMeter(),
 		ResolveNodeLimit: defaultResolveNodeLimit,
+		CheckResolver:    graph.NewLocalChecker(storage.NewContextualTupleDatastore(ds), 100),
```

Comment:
can you create a constant for the 100? it's being set in a few places now

Reply:
Done.